README.md

```diff
@@ -7,37 +7,84 @@
 
 Deployment
 ==========
-1. Create a new **Python 3.6.7** environment.
-2. Install project docker config generator requirements:
+1. Create a new **Python 3.6.7** virtual environment.
+1. Install requirements for Docker config generator:
     ```bash
     pip -r install gen_requirements.txt
     ```
-3. Install and configure [Docker](https://docs.docker.com/install/) and [Docker-compose](https://docs.docker.com/compose/install/)
-4. Set up an environmental variable for storing high volume downloadable data, like pre-trained models in [.env](../.env) file.
-``EXTERNAL_FOLDER=''`` - this variable stores path for external folder where downloaded data will be stored. By default it is linked to project's homefolder.
-5. Configure `TELEGRAM_TOKEN` and `TELEGRAM_PROXY` environment variables if you want to communicate with the bot via Telegram.
-6. Configure all skills, skill selectors, response selectors, annotators and database connection in [config.py](core/config.py)
-7. Generate **docker-compose.yml** by running the command:
+1. Install and configure [Docker](https://docs.docker.com/install/) and [Docker-compose](https://docs.docker.com/compose/install/) (version 1.19.0 or later).
+
+1. Create a directory for storing downloaded data, such as pre-trained models.
+   It should be located outside the agent project's home directory.
+   
+1. Create a `.env` file in the agent's home directory.
+   Add an `EXTERNAL_FOLDER` variable with the path to data directory:
+   
+   ```dotenv
+   EXTERNAL_FOLDER=<path to data directory>
+   ```
+1. If you want to communicate with the bot via Telegram, add the following environment variables to `.env`:
+
+   ```dotenv
+   TELEGRAM_TOKEN=<token>
+   TELEGRAM_PROXY=socks5://<user>:<password>@<path:port>
+   ```
+   
+   Here's an example of values:
+   
+   ```dotenv
+   TELEGRAM_TOKEN=123456789:AAGCiO0QFb_I-GXL-CbJDw7--JQbHkiQyYA
+   TELEGRAM_PROXY=socks5://tgproxy:tgproxy_pwd@123.45.67.89:1447
+   ```
+
+1. To enable using GPU, add the following variable to `.env`:
+
+   ```dotenv
+   CUDA_VISIBLE_DEVICES=<number of available GPU cores>
+   ```
+   
+1. Configure all skills, skill selectors, response selectors, annotators and database connection in [core/config.py](core/config.py).
```

Comment:
We need a separate README chapter with config description. Now it's cryptic how to configure services and database :)